### DISCUSSION TITLE
Embedding HatTip into a meta framework

### DISCUSSION THREAD

**Discussion by brillout** [hattipjs/hattip#167] [General]
I've created a React meta framework on top of VPS: https://github.com/brillout/restack and I've been looking at Rakkas's code for how to use HatTip for Restack. At first sight it doesn't seem that easy, although I'll have a closer look at it.

If there is interested, I'm happy to contribute to the design of the interface and document it.

**Comment by cyco130** [comment]
[`feat/cli`](https://github.com/hattipjs/hattip/tree/feat/cli) branch probably has everything (it's basically Rakkas CLI rebranded). Let me clean it up so we avoid duplicate effort.
